website/docs/add-secure-apps/applications/manage_apps.md

```diff
@@ -20,13 +20,21 @@ To add an application to authentik and have it display on users' **My applicatio
 
 2. Click **Create with Wizard**. (Alternatively, use our legacy process and click **Create**. The legacy process requires that the application and its authentication provider be configured separately.)
```

Comment:
Not part of this PR, but would it providing a link to the "legacy process" in its entirety? We have that as a User Story, right?  [Worried Princess Amadala Meme.]. Right?

Reply:
Sorry to the Princess, but I don't think we ever documenting the soon-to-be-legacy way. I was holding out for the Wizard. (But in principle I agree!)